express-zod-api/src/documentation.ts

```
@@ -86,6 +86,14 @@ interface DocumentationParams {
    * @example { users: "About users", files: { description: "About files", url: "https://example.com" } }
    * */
   tags?: Parameters<typeof depictTags>[0];
+/**
```

ESLint

[error] Insert `··`

(prettier/prettier)